packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -366,10 +366,14 @@ abstract class ScrollView extends StatelessWidget {
   final DragStartBehavior dragStartBehavior;
 
   /// {@template flutter.widgets.scroll_view.keyboardDismissBehavior}
-  /// [ScrollViewKeyboardDismissBehavior] the defines how this [ScrollView] will
+  /// [ScrollViewKeyboardDismissBehavior] defines how this [ScrollView] will
   /// dismiss the keyboard automatically.
   /// {@endtemplate}
-  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  ///
+  /// If [keyboardDismissBehavior] is null then it will fallback to
```

Comment:
What do you mean exactly ? I didn't understand who the flutter doc / template works and that so time consumming.

Must i copy / paste the bellow line of code under all usages of `{@macro flutter.widgets.scroll_view.keyboardDismissBehavior}`  ? That will duplicated documentation and didn't take power of the doc macro system... 

```dart
/// If [keyboardDismissBehavior] is null then it will fallback to
/// [scrollBehavior] or to the closest [ScrollConfiguration] provided
/// by the context.
```

Reply:
The {@endtemplate} is where the template ends. I was noting the docs you left outside of the template, meaning they would not be propagated in the other places where the template is used.

Yes, the purpose of templates is to avoid duplicating documentation, but the way you have written this excludes the information from the template. I assumed this was intentional, since you would get a doc resolution error if you did, because [scrollBehavior] is not a valid member of the other classes.